api/features/workflows/core/models.py

```diff
@@ -170,3 +199,12 @@ def send_email_notification_to_author(self):
                 context,
             ),
         )
+        message = CHANGE_REQUEST_APPROVED_MESSAGE % self.change_request.title
```

Comment:
As above, I think this would be better off in a hook?

Reply:
done